packages/clay-drop-down/src/Item.tsx

```diff
@@ -28,6 +28,13 @@ interface IProps
 
 	innerRef?: React.Ref<any>;
 
+	/**
+	 * Flag to indicate that the element is a session, are for cases when
+	 * Checkbox and Radio do not need the styles when hovering over the
+	 * drop-down item.
+	 */
+	isSection?: boolean;
```

Comment:
I'm not really sure about that name, I thought about maybe just leaving `section`. Suggestions?

Reply:
Yeah I think just `section` is a better fit for this, unless we want to create a new "Section" component for dropdown, but I think just a flag would be enough.

Reply:
Hmm, I think a new component for "Section" makes sense, but in order not to duplicate it we would have to have an ItemBase to reuse the logic. What do you think? I think I like to have a new component for that even though what will only change is the className 😅

Reply:
@matuzalemsteles new component is fine with me. Any idea if we need to support the icon for a section? If not, we wont need to duplicate too much

Reply:
@bryceosterhaus I don't know but we can go without the icon support for now.